File: Best-Time-to-Buy-and-Sell-Stock-II.py
class Solution(object):
    def maxProfit(self, p):
        \\\
        :type prices: List[int]
        :rtype: int
        \\\
        maxp = 0
        psize = len(p)

        for i in range (1, psize):
            if p[i] > p[i-1]:
                maxp += p[i] - p[i-1]

        return maxp

File: Best-Time-to-Buy-and-Sell-Stock-II.cpp
class Solution 
{
public:
    int maxProfit(vector<int>& p) 
    {
        int maxp = 0;
        int psize = p.size();

        for (int i = 1; i < psize; i++) 
        {
            if (p[i] > p[i - 1]) 
            {
                maxp += p[i] - p[i - 1];
            }
        }

        return maxp;
    }
};
